talent/onboarding/onboarding.md

```diff
@@ -0,0 +1,68 @@
+
+## Onboarding Process
+
+- This document describes the onboarding process for a new source{d} member.
+
+### Guide 
+
+- Read the guide at https://github.com/src-d/guide. The source{d} guide contains all the information necessary from the moment you accept an offer. 
+
+### Hardware
+
+- A few days before you start working with us, you choose the hardware you would like to use from on a list with the hardware we provide. 
```

Comment:
Let's add a link to https://github.com/src-d/guide/blob/master/general/available_hardware.md i.e by wrapping `from the list with available hardware` with `[](../../general/available_hardware.md)`